pytorch_lightning/loggers/base.py

```diff
@@ -305,6 +305,12 @@ def name(self) -> str:
     def version(self) -> Union[int, str]:
         """Return the experiment version."""
 
+    def _add_prefix(self, metrics: Dict[str, float], prefix: str, join_char: str):
```

Comment:
Any reason why `prefix` and `join_char` is given as function arguments when they are part of self?

Reply:
no reason. Updated. Thanks.